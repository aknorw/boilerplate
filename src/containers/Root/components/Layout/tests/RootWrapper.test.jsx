import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import RootWrapper from '../RootWrapper'

describe('<RootWrapper />', () => {
  it('should be defined', () => {
    expect(RootWrapper).toBeDefined()
  })
  it('should render correctly', () => {
    const tree = shallow(
      <RootWrapper />,
    )
    expect(tree).toMatchSnapshot()
  })
  it('should have style rules', () => {
    const tree = shallow(
      <RootWrapper />,
    )
    expect(tree).toHaveStyleRule('display', 'flex')
    expect(tree).toHaveStyleRule('flex-direction', 'column')
    expect(tree).toHaveStyleRule('min-height', '100vh')
    expect(tree).toHaveStyleRule('overflow-x', 'hidden')
  })
})
