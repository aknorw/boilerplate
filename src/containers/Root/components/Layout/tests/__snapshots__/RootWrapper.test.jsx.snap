// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RootWrapper /> should render correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  min-height: 100vh;
  overflow-x: hidden;
}

<div
  className="c0"
/>
`;
